fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -42,15 +41,13 @@
     **/
     public static final String REPOSITORY_NAMESPACE = "http://fedora.info/definitions/v4/repository#";
 
-    public static final String EVENT_NAMESPACE = "http://fedora.info/definitions/v4/event#";
-
-    public static final String FCREPO_API_NAMESPACE = "http://fedora.info/definitions/fcrepo#";
+    private static final String FCREPO_API_NAMESPACE = "http://fedora.info/definitions/fcrepo#";
```

Comment:
I'm a little unsure about changing the visibility of these constants. I understand they aren't used currently outside this class, so its probably fine, it just seems likely to get flipped back later when we need to use one of these namespaces in a test or something.